lesson-05-time-management/usrmod.c

```diff
@@ -0,0 +1,108 @@
+#include <linux/module.h>
+#include <linux/fs.h>
+#include <linux/cdev.h>
+#include <asm/uaccess.h>
+#include <linux/pci.h>
+#include <linux/version.h>
+#include <linux/init.h>
+#include <linux/timer.h>
+#include <linux/jiffies.h>
+#include <linux/kernel.h>
+
+#include "fib.c"
```

Comment:
Including source file?
Are you sure? Haven't you meant to do that via makefile?